main.py

```diff
@@ -51,10 +53,22 @@ def end_session(self, session_token: str) -> None:
 
     def get_customer(self, account_number: str) -> Union[Customer, None]:
         for customer in self.customers:
-            if customer.account.account_number == account_number:
-                return customer
+            for account in customer.accounts:
+                if account.account_number == account_number:
+                    return customer
         return None
 
+    def print_customers_info(self) -> None:
+        for customer in self.customers:
+            for account in customer.accounts:
+                print(f"Account Number: {account.account_number}")
+                print(f"Balance: ${account.balance}")
+                print(f"Is Locked: {account.is_locked}")
+                print("Transactions:")
+                for transaction in account.transactions:
+                    print(f"\t{transaction}")
+                print()
+
```

Comment:
I think it would be better to just override str in the Customer and Account classes which would not require to have the print formatting here